doc/object.types.propdefinition.md

```diff
@@ -0,0 +1,4 @@
+@typedef {Object|String|Constructor|Array|GETTER|SETTER} can-define-object/object.types.propDefinition PropDefinition
+@parent can-define-object/object.types
+
+STuff here
```

Comment:
This a WIP? Or was it supposed to be removed?

Reply:
Yep, WIP